### ISSUE TITLE
So, is OOC development still onging?

### ISSUE THREAD

**Issue by ghost** [fasterthanlime/rock#1006]
Hello good people,
Is this thing on? Is OOC/Rock development still happening?
Really love the language, would love to see it skyrocket. 
Please tell me it's still happening...

**Comment by horasal** [comment]
AFAIK, https://github.com/magic-lang/rock/tree/rock_1.0.22 is currently active.

**Comment by ghost** [comment]
So, it's a fork from ooc-lang github account...

**Comment by alexnask** [comment]
Hi, I plan on occasionally fixing bugs but ooc is pretty much in feature freeze and I don't have as much time as I used to to work on it.  
To be honest, the compiler rewrite should have happened long ago and that  would have probably kept me going but I never quite found the time to do it myself.

The magic-lang fork is still going strong and I would be happy to contribute features from time to time.  
@adrianvoica Thank you for your interest :)

**Comment by fasterthanlime** [comment]
Yup, that's about right. magic-lang is an ooc fork and is being used actively by some companies who develop it. The original ooc is on indefinite hiatus — I'm still around, but am not using it for anything nowadays and don't really have time to pursue it as a fun side project.

**Comment by ghost** [comment]
Seems fair. We all have our money-making priorities sooner or later that make us leave aside our hobbies.
Well, I wish you all greener pastures! 
Cheers, and great job!
